tst/EnergyPlus/unit/DOASEffectOnZoneSizing.unit.cc

```diff
@@ -139,7 +139,7 @@ TEST_F(EnergyPlusFixture, DOASEffectOnZoneSizing_CalcDOASSupCondsForSizing)
 
 TEST_F(EnergyPlusFixture, DOASEffectOnZoneSizing_SizeZoneEquipment)
 {
-
+    state->init_state(*state);
```

Comment:
Hmm...why does this one specific unit test need this added?  Maybe it doesn't since you added it to EnergyPlusFixture?  And we should delete this line?  If not, I'm just curious why it's needed here and not in every other test.

Reply:
After 2 days of unit test updates and trying different things to find the easiest path I'll have to check if this one and others you point out are sill needed.